pallets/subtensor/src/benchmarks.rs

```diff
@@ -429,30 +429,4 @@ reveal_weights {
 
   }: reveal_weights(RawOrigin::Signed(hotkey.clone()), netuid, uids, weight_values, salt, version_key)
 
-  schedule_coldkey_swap {
```

Comment:
I see no harm in leaving the benchmark in here, unless it creates compile errors? Benchmarking stuff is already `#[cfg(test)]` internally (I am 90% sure of this haven written the benchmarking v2 syntax at parity ;) ) so that should be 100% compatible with your `#[cfg(test)]` change unless there is some additional issue with how the weights get propagated through the macros that I'm not foreseeing (possible, but you'd get a compile error)

Reply:
Its always the 10% that kills you ;)

Reply:
though I wonder if you cfg test that as well if that would have fixed it 🤔